### DISCUSSION TITLE
File necessari

### DISCUSSION THREAD

**Discussion by bulltech-informatica** [LukeSavefrogs/danea-easyfatt#139] [Q&A]
ho scaricato l'installer ma non riesco a capire dove trovare i file necessari per l'installazione

[28-10-2023 11:14:10] CRITICAL File richiesto 'C:\Users\Administrator\Downloads\Documenti.DefXml' non        main.py:46
                               trovato.

**Comment by LukeSavefrogs** [comment]
Ciao @bulltech-informatica,
ora come ora il programma _RICHIEDE un file `.DefXml`_ (risultato dell'esportazione Documenti cliente da Easyfatt) nella cartella specificata dalla voce `files.input.easyfatt` nella configurazione.

Se hai questo file ma è chiamato in un altro modo o si trova in un'altra posizione quello che devi fare è utilizzare un file di configurazione personalizzato ([qui](https://lukesavefrogs.github.io/danea-easyfatt/utente/configurazione.html) puoi trovare la documentazione).

Questo file (chiamato `veryeasyfatt.config.toml`) va posizionato nella stessa cartella dell'eseguibile e deve essere configurato in base alle proprie necessità.

Come default usa dei valori "ragionevoli" che possano andare bene un pò a tutti: nel tuo caso ad esempio cerca il file `.DefXml` all'interno della stessa cartella dell'eseguibile. La voce che dovresti modificare tu è [`files.input.easyfatt`](https://lukesavefrogs.github.io/danea-easyfatt/utente/configurazione.html#filesinputeasyfatt).

**Cosa stai cercando di fare in particolare?**

**Reply by LukeSavefrogs** [reply]
Effettivamente i file obbligatori sono un **refuso** delle versioni prima che venisse aggiunto il **menu di scelta**, quindi IN TEORIA dovrebbe poter essere possibile rimuoverli senza molti problemi (ovviamente il controllo verrebbe effettuato lo stesso quando si sceglie una funzione che usi quei file).

**EDIT**: Creato issue #140 per questa modifica

**Comment by LukeSavefrogs** [comment]
> ho scaricato l'installer ma non riesco a capire dove trovare i file necessari per l'installazione
> 
> [28-10-2023 11:14:10] CRITICAL File richiesto 'C:\Users\Administrator\Downloads\Documenti.DefXml' non main.py:46 trovato.

Se non hai altro da aggiungere chiudo la discussione

**Comment by LukeSavefrogs** [comment]
Chiudo discussione per mancata risposta del richiedente
